components/CameraGallery.js

```diff
@@ -15,27 +23,95 @@ export default class CameraGallery extends Component {
       mediaTypes: ImagePicker.MediaTypeOptions.Images,
       allowsEditing: true,
     });
-    this.setState({ image: result.uri });
+    if  ((result != this.state.testImage) && (this.state.counter == 0)) {
+      this.setState({ image1: result.uri });
+      this.state.counter = 1;
+     }
+    else if ((result != this.state.testImage2) && (this.state.counter == 1)) {
+      this.setState({ image2: result.uri });
+      this.state.counter = 2;
+     }
+     else if ((result != this.state.testImage3) && (this.state.counter == 2)) {
+      this.setState({ image3: result.uri });
+      this.state.counter = 3;
+     }
+     else if ((result != this.state.testImage4) && (this.state.counter == 3)) {
+      this.setState({ image4: result.uri });
+      this.state.counter = 4;
+     }
+     else if ((result != this.state.testImage5) && (this.state.counter == 4)) {
+      this.setState({ image5: result.uri });
+      this.state.counter = 5;
+     }
+     else if ((result != this.state.testImage6) && (this.state.counter == 5)) {
+      this.setState({ image6: result.uri });
+      this.state.counter = 6;
+     }
   };
 
 // retrieves an image from a gallery
-  pickImage= async () => {
+  pickImage= async () => { 
     let result = await ImagePicker.launchImageLibraryAsync({
       mediaTypes: ImagePicker.MediaTypeOptions.Images,
       allowsEditing: true,
      });
-    this.setState({ image: result.uri });
+     
+     if  ((result != this.state.testImage) && (this.state.counter == 0)) {
+      this.setState({ image1: result.uri });
+      this.state.counter = 1;
+     }
+    else if ((result != this.state.testImage2) && (this.state.counter == 1)) {
+      this.setState({ image2: result.uri });
+      this.state.counter = 2;
+     }
+     else if ((result != this.state.testImage3) && (this.state.counter == 2)) {
+      this.setState({ image3: result.uri });
+      this.state.counter = 3;
+     }
+     else if ((result != this.state.testImage4) && (this.state.counter == 3)) {
+      this.setState({ image4: result.uri });
+      this.state.counter = 4;
+     }
+     else if ((result != this.state.testImage5) && (this.state.counter == 4)) {
+      this.setState({ image5: result.uri });
+      this.state.counter = 5;
+     }
+     else if ((result != this.state.testImage6) && (this.state.counter == 5)) {
+      this.setState({ image6: result.uri });
+      this.state.counter = 6;
+     }
   };
-
+   
+// renders the six images
   render() {
-    let { image } = this.state;
+     let { image1 } = this.state;
+     let { image2 } = this.state;
+     let { image3 } = this.state;
+     let { image4 } = this.state;
+     let { image5 } = this.state;
+     let { image6 } = this.state;
```

Comment:
In the future, we'll want to see if we can make this more efficient without having to copy and paste things. This works for now! Nice